### PR TITLE
fix: bicep config file update

### PR DESCRIPTION
**Pull request by wenytang-ms**
E2E test: https://github.com/OfficeDev/TeamsFx/actions/runs/1759533259
fix issue: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/13118546

### CODE REVIEW

**Comment by wenytang-ms** [comment]
E2E Test : https://github.com/OfficeDev/TeamsFx/actions/runs/1759533259

File: packages/fx-core/src/plugins/solution/fx-solution/arm.ts [hunk 1]
```diff
@@ -1023,39 +1023,39 @@ async function persistBicepTemplates(
...
-      bicepOrchestrationConfigContent = await fs.readFile(
-        path.join(getTemplatesFolder(), "plugins", "solution", "config.bicep"),
-        ConstantString.UTF8Encoding
-      );
-      const mainConfig = await fs.readFile(
-        path.join(templateSolitionPath, bicepOrchestrationConfigMainFileName),
-        ConstantString.UTF8Encoding
-      );
+    // If no any config part in orchestration or module, return.
+    if (bicepOrchestrationTemplate.getOrchestractionConfigContent() != "") {
```
Comment (blackchoey) [inline packages/fx-core/src/plugins/solution/fx-solution/arm.ts:1027]: `!==`?

File: packages/fx-core/src/plugins/solution/fx-solution/arm.ts [hunk 2]
```diff
@@ -1023,39 +1023,39 @@ async function persistBicepTemplates(
...
-      await fs.ensureDir(path.join(templateFolderPath, "teamsFx"));
-      bicepOrchestrationConfigContent = await fs.readFile(
-        path.join(getTemplatesFolder(), "plugins", "solution", "config.bicep"),
-        ConstantString.UTF8Encoding
-      );
-      const mainConfig = await fs.readFile(
-        path.join(templateSolitionPath, bicepOrchestrationConfigMainFileName),
-        ConstantString.UTF8Encoding
-      );
+    // If no any config part in orchestration or module, return.
```
Comment (dilin-MS) [inline packages/fx-core/src/plugins/solution/fx-solution/arm.ts:1026]: The comment does not match the code.

**Review by dilin-MS — APPROVED** [review]

File: packages/fx-core/src/plugins/solution/fx-solution/arm.ts (hunk 1 above)
Comment (wenytang-ms) [reply packages/fx-core/src/plugins/solution/fx-solution/arm.ts]: updated. thanks!

File: packages/fx-core/src/plugins/solution/fx-solution/arm.ts (hunk 2 above)
Comment (wenytang-ms) [reply packages/fx-core/src/plugins/solution/fx-solution/arm.ts]: updated.

**Review by blackchoey — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**